golem-templates/templates/c/c-app-common/wit/common.wit

```diff
@@ -1 +1,7 @@
+// This file is used to make the 'wit' directory compatible with tools such as wit-deps.
+//
+// Do NOT add anything to file, rather create a 'deps' directory, and place common WIT dependency packages there.
```

Comment:
to this file

Reply:
https://github.com/golemcloud/golem-cli/pull/153/commits/ca49a3b88ba9f72ca1e144338378cf76ad9a9427